### PR TITLE
Text Improvements to step-by-step chapter: scenes and nodes

### PR DESCRIPTION
**Pull request by Sceluswe**
- Improve introduction and remove verbose text.
- Add clarifications to some sentences.
- Add a new and shorter example.
- Add spaceship.png image to help readers understand the new example.
- Remove verbose text from the tutorial content.
- Improve existing tree.png image (it will no longer look stretched and unreadable).
- Replace side note text with an actual note.
- Structure and change the text to flow naturally from one premise to the next.

### CODE REVIEW

File: getting_started/step_by_step/scenes_and_nodes.rst
Comment (akien-mga) [inline getting_started/step_by_step/scenes_and_nodes.rst]: We're proud of our documentation having a few light-hearted sections (which are actually very rare), and the overwhelming majority of users who do comment about it seem to enjoy it too.

So this is not going away. And definitely not without discussion in a PR with 45 commits.

**Review by akien-mga — CHANGES_REQUESTED** [review]

File: getting_started/step_by_step/scenes_and_nodes.rst
Comment (Sceluswe) [reply getting_started/step_by_step/scenes_and_nodes.rst]: There should only be 15 commits (the ones pushed 2020-02-05). Not sure why the previous 30 got added to this PR since they were already merged with your master and are no longer active in my fork, they contain grammar fixes. 

In my opinion verbose text doesn't belong in a tutorial and/or documentation. I understand the purpose is to add something light-hearted but for me it just got in the way. I'm there to learn an already complex concept with a technical explanation so I can start making games. Even though it is light-hearted (even a little whimsical) it only added another layer to unpack instead of hitting premise after premise like a numbered list, which is what I think an effective tutorial should do. It added 3 paragraphs to the tutorial that I had to read in order to decide I could ignore them completely. 

With that said, you're the boss, and I'll make sure it's in there as soon as I find the time.

**Comment by Calinou** [comment]
This should be good to merge after solving merge conflicts. Please [rebase your pull request](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase) to solve them :slightly_smiling_face:

**Comment by vesk4000** [comment]
This looks like a really good and useful change! I hope it gets merged soon!

**Pull request closed without merging**
